index.js

```diff
@@ -164,4 +174,27 @@ form.addEventListener('submit', (e) => {
       setTimeout(() => errorPara.classList.remove('show-error'), 2000);
     }
   }
-});
\ No newline at end of file
+});
+/** * ---- End of Contact form validation functionality ---- ** */
+
+/** * ---- Start of Local Storage Functionality ---- ** */
+const detailObj = { name: '', email: '' };
```

Comment:
Great job storing the name and email inputs but as required by the project requirements, I kindly suggest that you also store the message input.

<img width="552" alt="image" src="https://github.com/Akorede3133/portfolio/assets/52934128/aedd82ad-cce9-4793-b4fd-e39abde4df68">

Your Storage Object is missing the message input:
<img width="256" alt="image" src="https://github.com/Akorede3133/portfolio/assets/52934128/36dd094b-c899-4587-856b-369f98b032c8">